src/aind_registration_evaluation/util/cellpose_util.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
This notebook must not be please here, it is necessary or were they just experiments?